app/models/repair.rb

```diff
@@ -4,9 +4,7 @@ def initialize(repair_data)
   end
 
   def request_reference
-    @repair_data.fetch('requestReference') do |_key|
```

Comment:
There's a spec for the second case here which needs deleting (or at least amending) - the context is `'when there is a reference with an outdated key'`

Reply:
Good spot! I'll remove that old test.